### DISCUSSION TITLE
Firefox Version

### DISCUSSION THREAD

**Discussion by carloswm85** [InternetConnectionMonitor/icm-public#85] [Ideas and Feature requests]
Is there a Firefox Version for this extension? I **have to** use Firefox at my job, and I want this extension on it.

**Comment by dmishh** [comment]
There is no Firefox version available at this point, but I'll keep this ticket to track the request to build it

**Comment by dougharris** [comment]
The frameworks for Chrome & Firefox extensions are very similar now. I have an extension that I build for both. [Here's the script I use](https://github.com/dougharris/unified_docs_switcher/blob/main/package.sh) to build. It's pretty specific to my extension but I think it's pretty well commented and might help you.

**Comment by Pikcee** [comment]
There is nothing similar to this extension on Firefox and I do really need it, if someone could help port this I'd be really grateful.

**Comment by ajaytalwar** [comment]
It would really be helpful to have a Firefox extension. We primarily use Firefox in our office (we don't trust Google!!!) and, being in a somewhat backwoods location, do have connectivity issues at times. Having this would really help. Had to install Opera to use  your extension. A Firefox version would be much appreciated. Thanks so much!

**Comment by reallybeingme** [comment]
I will add my vote to please make a firefox extension. I came across this extension on redditt.

**Comment by snjvm** [comment]
I will add my vote to please make a Firefox extension. It would really be helpful to have a Firefox extension. I primarily use Firefox in our office (they insist on Google Chrome / Edge, which I don't trust) and they don't permit Brave either, do have connectivity issues at times. Having this would really help. A Firefox version would be much appreciated. Thanks so much.

**Comment by brettles84** [comment]
please, please, please yes to a firefox version.
i LOVE this extension, just not enough to have chrome as my main browser these days

**Comment by pixelcruz** [comment]
hello, adding a +1 for a Firefox version, please ^_^
and thank you Dmytro for creating the chrome version!

**Comment by Kendeue** [comment]
The extension for Chrome is great. I would love this on both Firefox, Brave and Vivaldi aswell.

**Reply by dmishh** [reply]
@Kendeue you can already install ICM on Brave or any other Chromium-based browser

**Comment by 63787563** [comment]
Pretty please! If there's incentive needed I can make pretty pictures for things. Glad to offer my skills if there's a use.

**Comment by Dragonissa** [comment]
With Chrome/Chromium getting worse this is needed badly.

**Reply by rmw388** [reply]
I agree with this
